docs/pdr_avhw.md

```diff
@@ -0,0 +1,619 @@
+# Flight Hardware
+
+Lead: Matthew Glazer
+
+[TOC]
```

Comment:
This table of contents is suuuper long. Maybe manually pick out a few links to the top level items and place it here instead.

Reply:
is it better now?